Documentation/Contributors/TestingGuide/README.md

```diff
@@ -47,76 +39,6 @@ All new code should have 100% code coverage and should pass all tests. Always ru
     - [Debugger-Aided Incremental Improvements](#debugger-aided-incremental-improvements)
   - [Resources](#resources)
 
-## Running the Tests
```

Comment:
This should be fixed in [055a868](https://github.com/CesiumGS/cesium/pull/10184/commits/055a86868dcf23e352d4fc537fb6d52a712aeab3), but there are a few discrepancies from the old SpecRunner:

- There isn't a gulp task for running tests with the combined, unminified Cesium.js, but this was possible with the SpecRunner.
- Running `npm run test-release` doesn't work with `npm run combineRelease` either, because the "test" task looks for Cesium.js in `Build/Cesium/Cesium.js` and `combineRelease` puts Cesium.js in `Build/CesiumUnminified/`.

Reply:
That's fine, as long as we can continue to run the tests against release. It may be a bit more difficult to debug against minified code, but with build systems updates coming up I wouldn't get too hung up on running against `combinedRelease` yet. I think the adjustments you have in 055a868 make sense.